hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/GraphIndexTransaction.java

```diff
@@ -1350,22 +1338,22 @@ private static ConditionQuery constructQuery(ConditionQuery query,
         String joinedValues;
         // 2.1 All fields have equal-conditions
         if (prefixes.size() == fields.size()) {
-            // Prefix numeric values should be converted to sortable string
+            // Prefix numeric values should be converted to sort-able string
```

Comment:
why change this word?

Reply:
The IDE prompt "The word 'sortable' is not correctly spelled",  I will revert to "sortable".